upi/vsphere/ipam/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "name" {
+  type = string
+}
+
+variable "instance_count" {
+  type = string
+}
+
+variable "ignition" {
+  type    = string
+  default = ""
+}
+
+variable "cluster_domain" {
+  type = string
+}
+
+variable "machine_cidr" {
+  type = string
+}
+
+variable "ipam" {
+  type = string
+}
+
+variable "ipam_token" {
+  type = string
+}
+
+variable "ip_addresses" {
+  type = list(string)
+}
```

Comment:
this is not used.

since goal of ipam is to give you address, when we already have address, why would we want to execute the ipam?